eng/build.yml

```diff
@@ -10,12 +10,12 @@ steps:
 - script: dotnet build -c $(BuildConfiguration) --no-restore
   displayName: Build
 
-- script: dotnet pack -c $(BuildConfiguration) --no-build
-  displayName: Pack
-
 - task: DotNetCoreCLI@2
   displayName: Test
   inputs:
     command: test
     arguments: -c $(BuildConfiguration) --no-build /p:CollectCoverage=true /p:CoverletOutputFormat=opencover /p:Include=[coverlet.*]* /p:Exclude=[coverlet.tests.remoteexecutor]*
     testRunTitle: $(Agent.JobName)
+
+- script: dotnet pack -c $(BuildConfiguration)
```

Comment:
https://github.com/dotnet/sdk/issues/3001 new behaviour I think that pack will build core project.